charts/deployment/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.metrics.enabled }}
+apiVersion: monitoring.coreos.com/v1
```

Comment:
We will use hosted grafana so this needs to be changed to:
`apiVersion: azmonitoring.coreos.com/v1`